modules/openapi/component-protocol/scenarios/auto-test-scenes/components/apiEditor/render.go

```diff
@@ -157,12 +157,12 @@ LABEL:
 		if sStep.ID == stepID {
 			break
 		}
-		steps = append(steps, sStep)
 		for _, pStep := range sStep.Children {
 			if pStep.ID == stepID {
 				break LABEL
 			}
 		}
+		steps = append(steps, sStep)
```

Comment:
can you explain what logic in it?

Reply:
The old logic is to judge whether it is a parallel node first, and it should be a parallel node before attaching